0.64 API docs (#400)
microsoft/react-native-windows-samples: docs/IReactPackageBuilderExperimental-api-windows.md

Implements: [`IReactPackageBuilder`](IReactPackageBuilder)

> **EXPERIMENTAL**

Experimental extensions to the [`IReactPackageBuilder`](IReactPackageBuilder).

## Methods
### AddTurboModule
void **`AddTurboModule`**(string moduleName, [`ReactModuleProvider`](ReactModuleProvider) moduleProvider)

> Adds a custom TurboModule that uses direct access to JS Engine API.

Reviewer: ```suggestion
Adds a custom TurboModule that directly uses the JS Engine API.

```

Author: fixed
